cypress/integration/1221-my_publication/576-creator_cancels_deletion_of_an_item_in_my_registrations_list/cancel_delete_registration.js

```diff
@@ -0,0 +1,27 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_WITH_AUTHOR } from '../../../support/constants';
+
+// Feature: Creator cancels deletion of an item in My Registrations list
+
+// @576
+// Scenario: Creator cancels deletion of an item in My Registrations list
+Given('that the user is logged in as Creator', () => {
+  cy.login(USER_WITH_AUTHOR);
+});
+And('is on the My Registrations page', () => {
+  cy.get('[data-testid=my-registrations]').click({ force: true });
+});
+When('they click Delete on an item', () => {
+  cy.get('[data-testid^=registration-title]').first().as('registration');
+  cy.get('@registration')
+    .parent()
+    .within((registrationLine) => {
+      cy.get('[data-testid^=delete-registration]').click({ force: true });

```

Comment:
er ikke `cy.get('[data-testid^=delete-registration]').first().click({ force: true });` nok for denne When?